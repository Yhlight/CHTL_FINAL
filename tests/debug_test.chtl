div {
    style {
        height: 50% - 20%;
        padding: 10px % 3px;
    }
}
