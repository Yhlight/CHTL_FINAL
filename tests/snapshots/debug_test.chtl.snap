<div style="height: 30%; padding: 1px; " />
